src/app/netzgrafik-application/netzgrafik-application.component.ts

```diff
@@ -48,13 +52,24 @@ export class NetzgrafikApplicationComponent {
       .subscribe((params) => {
         uiInteractionService.setEditorMode(EditorMode.NetzgrafikEditing);
         uiInteractionService.showNetzgrafik();
-        versionControlService.load(params.getVariantId(), true);
+        try {
+          versionControlService.load(params.getVariantId(), true);
+        } catch (e) {
+          versionControlService.loadNetzgrafikDTO(NetzgrafikDefault.getDefaultNetzgrafik());
+        }
         uiInteractionService.setViewboxProperties(
           EditorView.svgName,
           uiInteractionService.getDefaultViewProperties());
       });
   }
 
+  getSidebarContainerStyle(): string {
+    if (this.disableBackend) {
+      return "top: 53px;";
+    }
+    return "top: 85px;";
+  }
```

Comment:
Hm, I wonder if we should instead add a `disable-backend` CSS class, and use that to guard the CSS rules, instead of this?